src/utils/common/txBuilder.ts

```diff
@@ -11,7 +11,7 @@ import {
   standardTransactionVersion,
 } from "./constants";
 
-export function buildTx({ ...args }: any) {
+export function buildTx({ ...args }) {
```

Comment:
this could still have a type consideration because i believe it's implying `any` and that implication could be bad ;)

Reply:
i'll happily kick this can down the road until there's more tx types to be built